packages/features/tasker/tasks/triggerNoShow/triggerGuestNoShow.ts

```diff
@@ -45,48 +48,58 @@ export async function triggerGuestNoShow(payload: string): Promise<void> {
   const {
     webhook,
     booking,
+    hosts,
     hostsThatJoinedTheCall,
     didGuestJoinTheCall,
     guestsThatDidntJoinTheCall,
     originalRescheduledBooking,
     participants,
   } = result;
 
+  const hostEmails = new Set(hosts.map((h) => h.email));
+
   const maxStartTime = calculateMaxStartTime(booking.startTime, webhook.time, webhook.timeUnit);
 
   const requireEmailForGuests = booking.eventType?.calVideoSettings?.requireEmailForGuests ?? false;
 
   if (requireEmailForGuests) {
     if (guestsThatDidntJoinTheCall.length > 0) {
-      await Promise.all([
-        sendWebhookPayload(
-          webhook,
-          WebhookTriggerEvents.AFTER_GUESTS_CAL_VIDEO_NO_SHOW,
-          booking,
-          maxStartTime,
-          participants,
-          originalRescheduledBooking
-        ),
-        markGuestAsNoshowInBooking({
-          bookingId: booking.id,
-          hostsThatJoinedTheCall,
-          guestsThatDidntJoinTheCall,
-        }),
```

Comment:
`markGuestAsNoshowInBooking` should be executed first to get the updated `booking.attendees`